services/harmony/app/middleware/shapefile-converter.ts

```diff
@@ -1,6 +1,9 @@
+/* eslint-disable require-jsdoc */
```

Comment:
We should fill in the JS docs for the functions.

Reply:
I disagree. This disable is here because there are a couple of utility functions that are defined internally to another function (something I wouldn't do myself, but chatGPT did). These functions don't represent any sort of API that needs documenting.

Reply:
The internal to a function ones are fine, but we should fill out the JS docs for the top level functions.

Reply:
Similar comment for the other lint ignores, I'd rather see explicit ignores above each line that at the top level of the file because we don't want to ignore JS docs for all the functions in this file, just the internal to a function ones if you want to keep them defined inside the function.

Reply:
Ok, I removed the top level linter ignore and added specific ones for the internal functions, along with JSDocs for the top-level ones missing them. I do with there was a way to tell VS Code to not underline everything in yellow just because I haven't written a JSDoc yet. It's annoying that it happens immediately when you create a function.